code/game/gamemodes/clock_cult/clock_mobs/_eminence.dm

```diff
@@ -20,7 +20,16 @@
 /mob/camera/eminence/Move(NewLoc, direct)
 	var/OldLoc = loc
 	if(NewLoc && !istype(NewLoc, /turf/open/indestructible/reebe_void))
-		forceMove(get_turf(NewLoc))
+		var/turf/T = get_turf(NewLoc)
+		if(T.flags_1 & NOJAUNT_1)
+			T.visible_message("<span class='warning'>[T] suddenly emits a ringing sound!</span>", ignore_mob = src)
```

Comment:
This seems like it can easily get spammy, maybe stun the eminence?

Reply:
#33605 will make it a non-issue.

Reply:
not necessarily, any intervening chat will make it appear again and the sound plays again as well